mappings/net/minecraft/server/world/ThreadedAnvilChunkStorage.mapping

```diff
@@ -1,38 +1,72 @@
+		COMMENT List of chunk positions that are not currently loaded.
+	FIELD field_17222 loadedChunksDirty Z
+	FIELD field_17223 prioritizationQueue Lnet/minecraft/class_3900;
 	FIELD field_17224 worldGenExecutor Lnet/minecraft/class_3906;
 	FIELD field_17226 mainExecutor Lnet/minecraft/class_3906;
 	FIELD field_17228 ticketManager Lnet/minecraft/class_3898$class_3216;
-	FIELD field_17230 totalChunksLoadedCount Ljava/util/concurrent/atomic/AtomicInteger;
-	FIELD field_17442 worldGenerationProgressListener Lnet/minecraft/class_3949;
+	FIELD field_17230 totalCompletedChunks Ljava/util/concurrent/atomic/AtomicInteger;
+		COMMENT The number of chunks that are completely finished loading. This value is incremented every time a chunk
+		COMMENT finished loading.
+	FIELD field_17442 worldGenProgressListener Lnet/minecraft/class_3949;
 	FIELD field_17705 persistentStateManagerFactory Ljava/util/function/Supplier;
 	FIELD field_17706 structureManager Lnet/minecraft/class_3485;
-	FIELD field_17707 saveDir Ljava/io/File;
+	FIELD field_17707 saveDirectory Ljava/io/File;
+		COMMENT The world directory itself.
 	FIELD field_18239 MAX_LEVEL I
 		COMMENT Specifies the maximum ticket level a chunk can be before a chunk's {@link net.minecraft.server.world.ChunkHolder.LevelType} is {@link net.minecraft.server.world.ChunkHolder.LevelType#BORDER}.
-	FIELD field_18241 playerChunkWatchingManager Lnet/minecraft/class_3210;
+	FIELD field_18241 chunkObservationTracker Lnet/minecraft/class_3210;
+		COMMENT Tracker used to determine whether a player is in and/or looking at a specific chunk.
+		COMMENT Chunks that are not being observed by anyone are more likely to be stored and unloaded.
 	FIELD field_18242 entityTrackers Lit/unimi/dsi/fastutil/ints/Int2ObjectMap;
-	FIELD field_18243 watchDistance I
-	FIELD field_18307 loadedChunks Lit/unimi/dsi/fastutil/longs/LongSet;
-	FIELD field_18807 chunksToUnload Lit/unimi/dsi/fastutil/longs/Long2ObjectLinkedOpenHashMap;
+	FIELD field_18243 chunkViewDistance I
+	FIELD field_18307 loadedPositions Lit/unimi/dsi/fastutil/longs/LongSet;
+		COMMENT Set of chunk positions that are currently loaded into memory.
+	FIELD field_18807 tickingChunks Lit/unimi/dsi/fastutil/longs/Long2ObjectLinkedOpenHashMap;
+		COMMENT The map of loaded chunks that also have ticking blocks/active tick operations.
 	FIELD field_18808 pointOfInterestStorage Lnet/minecraft/class_4153;
-	FIELD field_19343 unloadTaskQueue Ljava/util/Queue;
-	FIELD field_23786 chunkToType Lit/unimi/dsi/fastutil/longs/Long2ByteMap;
+	FIELD field_19343 unloadListeners Ljava/util/Queue;
+		COMMENT Event listeners triggered whenever a chunk is persisted to disk.
+	FIELD field_23786 loadedChunkStatuses Lit/unimi/dsi/fastutil/longs/Long2ByteMap;
+		COMMENT The status and type of all loaded chunk.
+		COMMENT Used to track the state of different chunks through the loading and saving phase.
+		COMMENT Typically, all thunks should either have a value of 0 or 1 (below). Chunks that fail to load
```

Comment:
Typo on `thunks`. Also, it’s not described what `1` means?

Reply:
> COMMENT <li>1 - loaded/level chunk</li>
It's the last one in the list, literally right under where you commented.

Reply:
Gosh, oops. I have no idea how I missed that. Sorry